eftcamb/11_EFTCAMB_stability.f90

```diff
@@ -143,101 +143,29 @@ function EFTStabilityComputation(a)
-!            &(-3._dl*EFTgpresdotQ -3._dl*adotoa*(EFTgrhoQ +EFTgpresQ)-3._dl*a*adotoa**3*(a*EFTOmegaPP +6._dl*EFTOmegaP) &
-!            & -6._dl*a*adotoa*Hdot*EFTOmegaP +3._dl*(Hdotdot-2._dl*adotoa*Hdot)*(EFTGamma4V +3._dl*EFTGamma3V)&
-!            & +6._dl*adotoa*(Hdot-adotoa**2)*(3._dl*EFTGamma3V +1.5_dl*a*EFTGamma3P +EFTGamma4V + 0.5_dl*a*EFTGamma4P)&
-!            & -3._dl*a*EFT_H0*(3._dl*adotoa**2*EFTGamma2V +Hdot*EFTGamma2V +a*adotoa**2*EFTGamma2P))&
-!            & +1._dl/(1._dl+EFTOmegaV+2._dl*EFTGamma5V)*(a*adotoa*EFTOmegaP+2._dl*adotoa*(EFTGamma5V +a*EFTGamma5P)&
-!            & -(1._dl+EFTOmegaV)*(a*adotoa*EFTOmegaP+a*EFT_H0*EFTGamma2V)/(2._dl*(1._dl+EFTOmegaV)+3._dl*EFTGamma3V +EFTGamma4V))*&
-!            &(-0.5*EFTgrhodotQ -adotoa*EFTc +1.5_dl*a*adotoa*EFTOmegaP*(3._dl*adotoa**2-Hdot) -2._dl*a2*adotoa*EFT_H0**2*EFTGamma1V&
-!            & -1.5_dl*a*EFT_H0*EFTGamma2V*(Hdot-2._dl*adotoa**2) -3._dl*adotoa*(Hdot-adotoa**2)*(1.5_dl*EFTGamma3V +0.5_dl*EFTGamma4V))
-!        ! First part of the D coefficient, the k^2 term:
-!        EFTpiD1 = EFTc -0.5_dl*a*adotoa*EFT_H0*(EFTGamma2V +a*EFTGamma2P) +(adotoa**2-Hdot)*(3._dl*EFTGamma3V +EFTGamma4V)&
-!            & +4._dl*(Hdot*EFTGamma6V + adotoa**2*EFTGamma6V + a*adotoa**2*EFTGamma6P)&
-!            & +2._dl*(Hdot*EFTGamma5V +a*adotoa**2*EFTGamma5P)&
-!            & -a*(adotoa*EFTOmegaP+EFT_H0*EFTGamma2V)/(4._dl*(1._dl+EFTOmegaV)+6._dl*EFTGamma3V +2._dl*EFTGamma4V)*&
-!            &(-2._dl*a*adotoa*EFTOmegaP +4._dl*adotoa*EFTGamma5V -2._dl*adotoa*(3._dl*EFTGamma3V +1.5_dl*a*EFTGamma3P &
-!            & +EFTGamma4V +0.5_dl*a*EFTGamma4P))&
-!            & +1._dl/(1._dl+EFTOmegaV+2._dl*EFTGamma5V)*(a*adotoa*EFTOmegaP+2._dl*adotoa*(EFTGamma5V +a*EFTGamma5P)&
-!            & -(1._dl+EFTOmegaV)*(a*adotoa*EFTOmegaP +a*EFT_H0*EFTGamma2V)/(2._dl*(1._dl+EFTOmegaV)+3._dl*EFTGamma3V +EFTGamma4V))*&
-!            &(+0.5_dl*a*adotoa*EFTOmegaP -2._dl*adotoa*EFTGamma5V +0.5_dl*a*EFT_H0*EFTGamma2V +1.5_dl*adotoa*EFTGamma3V&
-!            & +0.5_dl*adotoa*EFTGamma4V -4._dl*adotoa*EFTGamma6V)&
-!            & +(EFTGamma4V +2._dl*EFTGamma5V)/(2._dl*(1._dl+EFTOmegaV) -2._dl*EFTGamma4V)*(EFTgrhoQ +EFTgpresQ +a*adotoa**2*EFTOmegaP&
-!            & -EFTGamma4V*(Hdot-adotoa**2) +a*adotoa*EFT_H0*EFTGamma2V +3._dl*EFTGamma3V*(adotoa**2-Hdot))
-!        ! Second part of the D coefficient, the k^4 term:
-!        EFTpiD2 = +(+0.5_dl*EFTGamma3V +0.5_dl*EFTGamma4V &
-!            & +(EFTGamma4V +2._dl*EFTGamma5V)/(2._dl*(1._dl+EFTOmegaV) -2._dl*EFTGamma4V)*(EFTGamma3V +EFTGamma4V))
+!
+!        ! Compute pi field equations factors
+!        call CP%EFTCAMB%model%compute_pi_factors( a, CP%eft_par_cache , eft_cache )
+!        ! Compute coefficients for the tensor propagation equation
+!        call compute_tensor_factors( a, CP%eft_par_cache , eft_cache )
 !
 !        if ( CP%EFTflag==4 ) then ! Full mapping EFT overwrites some of these quantities:
```

Comment:
We can remove the Horava part entirely. Will be implemented later when Horava will be done.